tests_disabled.json

```diff
@@ -300,6 +300,10 @@
         {
             "name": "t_frang.test_concurrent_connections.ConcurrentConnections",
             "reason": "Disabled by test issue #525"
+        },
+        {
+            "name": "t_health_monitoring.test_health_monitor.H2HmResponsesPipelined",
+            "reason": "The test does not work on the SUT server - crash"
```

Comment:
We usually don't add tests to `tests_disabled*` files without a reference to an issue or PR. Please add the new issue to tempesta-test or tempesta repo, otherwise we will just forget that this test is disabled.

Reply:
I have added an issue #687

Reply:
And I have updated the description.